common/src/androidTest/java/com/microsoft/identity/common/internal/platform/DevicePoPManagerSigningTests.java

```diff
@@ -66,23 +55,7 @@
 
     @Parameterized.Parameters
     public static Iterable<SigningAlgorithm> testParams() {
-        final List<SigningAlgorithm> signingAlgs =
-                new ArrayList<SigningAlgorithm>() {{
-                    add(MD5_WITH_RSA);
-                    add(NONE_WITH_RSA);
-                    add(SHA_256_WITH_RSA);
-                    add(SHA_384_WITH_RSA);
-                    add(SHA_512_WITH_RSA);
-                }};
-
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-            // Only execute these tests at appropriate API levels...
-            signingAlgs.add(SHA_256_WITH_RSA_PSS);
-            signingAlgs.add(SHA_384_WITH_RSA_PSS);
-            signingAlgs.add(SHA_512_WITH_RSA_PSS);
-        }
-
-        return signingAlgs;
+        return new ArrayList<>(Arrays.asList(SigningAlgorithm.values()));
```

Comment:
nit: can we just return Arrays.asList here?

Reply:
14f411b2ceb6064e2b78219fab1a01284352c06f